models/repmaint.json

```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "repmaint_facts",
     "label": "Repairs and Maintenance",
-    "description": "This is based on the RME return form.",
+    "description": "Money spent on repairs and maintenance of assets.",
+    "return_forms": ["RME"],
```

Comment:
I'm beginning to think repmaint isn't useful, if we can get everything else from  capex.